package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "start": "parcel src/index.html --open -d public",
     "build": "parcel build src/index.html -d public",
-    "clean": "rm -rf public .cache"
+    "clean": "rm -rf public .cache",
+    "start:bot": "env-cmd babel-watch -w api/_bot -m '> Restarting bot!' api/_bot/polling-run.js"
```

Comment:
If somebody asks, `env-cmd` is to read and inject `.env` files. `babel-watch` is because the code uses ECMAScript modules which are not fully supported in node 13.x, but it also reloads when detects any change in the code.